src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetAnnotationsIntTest.java

```diff
@@ -170,6 +175,35 @@ void givenNonSuperAdminJudgeUserReturnForbidden() throws Exception {
             .andExpect(status().isForbidden())
             .andReturn();
 
+        verify(mockUserIdentity).getUserAccount();
+        verify(mockUserIdentity).userHasGlobalAccess(Set.of(JUDGE, SUPER_ADMIN));
+        verifyNoMoreInteractions(mockUserIdentity);
+    }
+
+    @Test
+    void givenSuperUserReturnForbidden() throws Exception {
```

Comment:
This should probably include forbidden for judges annotations

Reply:
I don't think it's needed as this integration test class is for the same endpoint 
`GET /cases/{case_id}/annotations` - Admin / Judge view for annotations
I just added this alongside the other test (givenNonSuperAdminJudgeUserReturnForbidden) just to have given a "Super User" identity to verify the response